src/program/programUtils.ts

```diff
@@ -1,7 +1,7 @@
 import type { BosKonfYear, Program } from "./program";
 
 export async function getProgram(year: BosKonfYear, language: "en" | "no") {
-  let json = await import(`./${year}.json`);
+  let json = await import(`./${year}.ts`);
```

Comment:
Why don't you import it directly from the files?

Reply:
To make it 🌟 dynamic ✨, but probably simpler to just import them one by one, we need to add another 2025.ts file next year either way so...

Reply:
Added static imports with clear instructions if it crashes